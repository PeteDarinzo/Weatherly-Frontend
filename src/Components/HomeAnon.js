import React from "react";
import { Link } from "react-router-dom";
import Typography from '@mui/material/Typography';
import ButtonGroup from '@mui/material/ButtonGroup';
import Button from '@mui/material/Button';
import Container from '@mui/material/Container';
import Box from '@mui/material/Box';
import Grid from '@mui/material/Grid';


/** Landing page for logged out users */

const HomeAnon = () => {


  return (
    <Container>
      <Grid container>
        <Grid item xs={12}>
          <Typography variant="h2" sx={{ margin: "20px" }}>WELCOME TO WEATHERLY</Typography>
        </Grid>
        <Grid item xs={6}>
          <Typography variant="h5" sx={{ marginTop: "10px", marginBottom: "10px", textAlign: "left" }}>What is it?</Typography>
          <Typography sx={{ textAlign: "left" }}>Your time is valuable.</Typography>
          <Typography sx={{ textAlign: "left" }}>Don't scroll endlessly looking for something to watch on a sunny day.</Typography>
          <Typography sx={{ textAlign: "left" }}>Keep your "to-watch" list close at hand.</Typography>
          <Typography sx={{ textAlign: "left" }}>Let the app find the next inclement weather day for you.</Typography>
        </Grid>
        <Grid item xs={12}>
          <Box
          // sx={{ width: "250px", backgroundColor: "" }}
          >
            <Typography
              sx={{ margin: "10px", textAlign: "left" }}>
              New to Weatherly?
              <Button
                component={Link}
                to="/signup"
                variant="outlined"
                sx={{ m: 2 }}
                color="primary"
              >Signup</Button>
            </Typography>
            <Typography
              sx={{ margin: "10px", textAlign: "left" }}>
              Already a user?
              <Button
                component={Link}
                to="/login"
                variant="outlined"
                sx={{ m: 2 }}
                color="secondary"
              >Login</Button>
            </Typography>
          </Box>
        </Grid>
      </Grid>
    </Container>
  )
}

export default HomeAnon;