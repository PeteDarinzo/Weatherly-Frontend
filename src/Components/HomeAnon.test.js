import HomeAnon from "./HomeAnon";
import { render } from '@testing-library/react';
import { MemoryRouter } from "react-router-dom"

it("renders home without crashing", () => {
  render(
    <MemoryRouter>
      <HomeAnon />
    </MemoryRouter>
  );
});

// it("matches the home snapshot", () => {
//   const { asFragment } = render(
//     <MemoryRouter>
//       <Home />
//     </MemoryRouter>
//   );
//   expect(asFragment()).toMatchSnapshot();
// });
