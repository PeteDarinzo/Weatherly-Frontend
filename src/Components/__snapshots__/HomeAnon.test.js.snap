// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the home snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
  >
    <div
      class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-1idn90j-MuiGrid-root"
      >
        <h2
          class="MuiTypography-root MuiTypography-h2 css-qq3owz-MuiTypography-root"
        >
          WELCOME TO WEATHERLY
        </h2>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 css-1osj8n2-MuiGrid-root"
      >
        <h5
          class="MuiTypography-root MuiTypography-h5 css-1jcyw1h-MuiTypography-root"
        >
          What is it?
        </h5>
        <p
          class="MuiTypography-root MuiTypography-body1 css-zpphrz-MuiTypography-root"
        >
          Your time is valuable.
        </p>
        <p
          class="MuiTypography-root MuiTypography-body1 css-zpphrz-MuiTypography-root"
        >
          Don't scroll endlessly looking for something to watch on a sunny day.
        </p>
        <p
          class="MuiTypography-root MuiTypography-body1 css-zpphrz-MuiTypography-root"
        >
          Keep your "to-watch" list close at hand.
        </p>
        <p
          class="MuiTypography-root MuiTypography-body1 css-zpphrz-MuiTypography-root"
        >
          Let the app find the next inclement weather day for you.
        </p>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-1idn90j-MuiGrid-root"
      >
        <div
          class="MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 css-fksj50-MuiTypography-root"
          >
            New to Weatherly?
            <a
              class="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root  css-19skcmy-MuiButtonBase-root-MuiButton-root"
              href="/signup"
              tabindex="0"
            >
              Signup
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </a>
          </p>
          <p
            class="MuiTypography-root MuiTypography-body1 css-fksj50-MuiTypography-root"
          >
            Already a user?
            <a
              class="MuiButton-root MuiButton-outlined MuiButton-outlinedSecondary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root  css-1do0yfw-MuiButtonBase-root-MuiButton-root"
              href="/login"
              tabindex="0"
            >
              Login
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </a>
          </p>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
